### PR TITLE
rosy: add i2c sensors

### PR DESCRIPTION
**Pull request by Tofee**
Add i2c sensors for gyroscope (bmi120), accelerometer (ak09918) and ALS (ltr579, no driver support yet).

### CODE REVIEW

File: arch/arm64/boot/dts/qcom/sdm450-xiaomi-rosy.dts [hunk 1]
```diff
@@ -31,6 +31,47 @@
...
+
+			vdd-supply = <&pm8953_l10>;
+			vid-supply = <&pm8953_l6>;
+		};
+
+		/*
+		 * Driver still missing
+		 * Just for general information
+		 */
+		ltr579: light@53 {
```
Comment (barni2000) [inline arch/arm64/boot/dts/qcom/sdm450-xiaomi-rosy.dts:55]: i am not sure it hase ltr579 variant, my rosy uses stk3x1x. There is an st3310 driver in the kernel.

Comment (Tofee) [reply arch/arm64/boot/dts/qcom/sdm450-xiaomi-rosy.dts:55]: I can try with st3310; I have no idea which one my rosy uses

Comment (barni2000) [reply arch/arm64/boot/dts/qcom/sdm450-xiaomi-rosy.dts:55]: You can check your device with smgr

File: arch/arm64/boot/dts/qcom/sdm450-xiaomi-rosy.dts [hunk 2]
```diff
@@ -31,6 +31,47 @@
...
+			compatible = "liteon,ltr579";
+			reg = <0x53>;
+
+			vdd-supply = <&pm8953_l10>;
+			vid-supply = <&pm8953_l6>;
+		};
+
+		bmi120: imu@68 {
```
Comment (barni2000) [inline arch/arm64/boot/dts/qcom/sdm450-xiaomi-rosy.dts:63]:
```suggestion
		imu@68 {
```

File: arch/arm64/boot/dts/qcom/sdm450-xiaomi-rosy.dts [hunk 3]
```diff
@@ -31,6 +31,47 @@
...
+			vdd-supply = <&pm8953_l10>;
+			vid-supply = <&pm8953_l6>;
+		};
+
+		/*
+		 * Driver still missing
+		 * Just for general information
+		 */
+		ltr579: light@53 {
+			compatible = "liteon,ltr579";
```
Comment (barni2000) [inline arch/arm64/boot/dts/qcom/sdm450-xiaomi-rosy.dts:56]: It has stk3311-a

File: arch/arm64/boot/dts/qcom/sdm450-xiaomi-rosy.dts [hunk 4]
```diff
@@ -31,6 +31,47 @@
...
+
+	i2c-sensors {
+		compatible = "i2c-gpio";
+		sda-gpios = <&tlmm 14 (GPIO_ACTIVE_HIGH|GPIO_OPEN_DRAIN)>;
+		scl-gpios = <&tlmm 15 (GPIO_ACTIVE_HIGH|GPIO_OPEN_DRAIN)>;
+		i2c-gpio,delay-us = <2>; /* ~100 kHz */
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		ak09918: magnetometer@c {
```
Comment (barni2000) [inline arch/arm64/boot/dts/qcom/sdm450-xiaomi-rosy.dts:43]:
```suggestion
		magnetometer@c {
```

**Review by barni2000 — CHANGES_REQUESTED** [review]

File: arch/arm64/boot/dts/qcom/sdm450-xiaomi-rosy.dts [hunk 5]
```diff
@@ -31,6 +31,47 @@
...
+		};
+
+		bmi120: imu@68 {
+			compatible = "bosch,bmi120";
+			reg = <0x68>;
+
+			vdd-supply = <&pm8953_l10>;
+			vddio-supply = <&pm8953_l6>;
+
+			mount-matrix = "0", "1", "0",
```
Comment (barni2000) [inline arch/arm64/boot/dts/qcom/sdm450-xiaomi-rosy.dts:70]: This is not correct.

**Review by barni2000 — CHANGES_REQUESTED** [review]

File: arch/arm64/boot/dts/qcom/sdm450-xiaomi-rosy.dts [hunk 6]
```diff
@@ -31,6 +31,47 @@
...
+					"1", "0", "0",
+					"0", "0", "1";
```
Comment (barni2000) [inline arch/arm64/boot/dts/qcom/sdm450-xiaomi-rosy.dts:72]:
```suggestion
			mount-matrix = "1", "0", "0",
					"0", "-1", "0",
					"0", "0", "1";
```
This is the correct mount-matrix

**Review by barni2000 — CHANGES_REQUESTED** [review]

**Comment by barni2000** [comment]
Change commit title prefixes to dts:qcom: xiaomi-rosy:

**Pull request merged**